pygg/__init__.py

```diff
@@ -1 +1,9 @@
+from ggplot2_functions import ggplot2_functions
+from pygg import make_ggplot2_binding
+
+for f in ggplot2_functions:
```

Comment:
I think this causes uses of `pygg.axis_labels` to break because scale_x_continuous isn't defined inside pygg.py.  One possibility is to move that function to __init__.py